zerver/openapi/zulip.yaml

```diff
@@ -8336,6 +8336,30 @@ paths:
         `DELETE {{ api_url }}/v1/users/{user_id}`
       parameters:
         - $ref: "#/components/parameters/UserId"
+        - name: email_text
+          in: query
+          description: |
+            The part of the email to the deactivated user that was written by the admin.
+          schema:
+            type: string
+          example: Bye user!
+          required: false
+        - name: realm_name
+          in: query
+          description: |
+            The name of the realm where the user gets deactivated.
+          schema:
+            type: string
+          example: Zulip Dev
+          required: false
+        - name: username
+          in: query
+          description: |
+            The name of the user that is getting deactivated.
+          schema:
+            type: string
+          example: Zoe
+          required: false
```

Comment:
As noted elsewhere, we only need the text field, which perhaps we can call `deactivation_notification_comment`?